chroma_agent/lib/corosync.py

```
@@ -367,6 +367,12 @@ def __getattr__(self, attr):
     def set_address(self, ipv4_address, prefix):
         ifaddr = "%s/%s" % (ipv4_address, prefix)
 
+        if not ipv4_address or not prefix:
+            console_log.info(
```

Should this be a runtime error?

---

I don't think so.  This sort of error was silently dropped configure_network()